src/engines/Cesium/core/Imagery.tsx

```diff
@@ -113,6 +113,12 @@ export function useImageryProviders({
   const tileKeys = tiles.map(t => t.id).join(",");
   const prevTileKeys = useRef(tileKeys);
   const prevProviders = useRef<Providers>({});
+  
+  const zoomLevels = tiles.map(t => {
+    if(t.id && t.zoomLevel) return {[t.id]: t.zoomLevel}
+    return
+  })
```

Comment:
Could you fix to use useMemo?
```suggestion
  const zoomLevels = useMemo(() => tiles.map(t => {
    if(t.id && t.zoomLevel) return {[t.id]: t.zoomLevel}
    return
  }), [tiles]);
```

Reply:
Thanks @keiya01 . This is done.